workshop_attendance.txt

```diff
@@ -1,4 +0,0 @@
-# Please enter your name and your github handle, each in a new line.
-
-Blanca Mancilla
-@bluciam
```

Comment:
Oh...Sorry.  I think I can revert it.


2015-10-08 20:37 GMT-04:00 Françoise Provencher <notifications@github.com>:

> In workshop_attendance.txt
> <https://github.com/bluciam/helloworld_pyladiesMTL/pull/39#discussion_r41587214>
> :
>
> > @@ -1,4 +0,0 @@
> > -# Please enter your name and your github handle, each in a new line.
> > -
> > -Blanca Mancilla
> > -@bluciam
>
> Oh no! You deleted the original file :-( Is it possible to revert the
> deletion of workshop_attendance.txt?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bluciam/helloworld_pyladiesMTL/pull/39/files#r41587214>
> .
>



-- 
molt.bandcamp.com